packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/cart-api/add-address.ts

```diff
@@ -0,0 +1,21 @@
+import {Tile, extension} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension('pos.home.tile.render', (root, api) => {
+  const tile = root.createComponent(Tile, {
+    title: 'Discount App',
+    subtitle: 'Apply cart discount',
```

Comment:
May want to change these to a generic title/subtitle so they're not misleading and then you don't have to adapt every single one to its context.